packages/engine/src/interaction/systems/MountPointSystem.ts

```diff
@@ -63,15 +65,32 @@ const mountPointInteractMessages = {
 }
 
 const mountPointActionQueue = defineActionQueue(EngineActions.interactedWithObject.matches)
+const mountPointSeatedQueue = defineActionQueue(MountPointActions.mountInteraction.matches)
 const mountPointQuery = defineQuery([MountPointComponent])
 const sittingIdleQuery = defineQuery([SittingComponent])
 
 const execute = () => {
+  for (const action of mountPointSeatedQueue()) {
```

Comment:
for networked actions, we should generally try to avoid action queues and use event sourcing instead